lib/matplotlib/tests/test_axes.py

```diff
@@ -6129,3 +6129,19 @@ def test_invisible_axes():
     assert fig.canvas.inaxes((200, 200)) is not None
     ax.set_visible(False)
     assert fig.canvas.inaxes((200, 200)) is None
+
+
+def test_xtickcolor_is_not_markercolor():
+    plt.rcParams['lines.markeredgecolor'] = 'white'
+    ax = plt.axes()
+    ticks = ax.xaxis.get_major_ticks()
+    for tick in ticks:
+        assert not tick.tick1line.get_markeredgecolor() == 'white'
```

Comment:
This should probably be:
```suggestion
        assert tick.tick1line.get_markeredgecolor() != 'white'
```

Reply:
Doesn't give a flake8 error for me, but I fixed it anyhow. Would you mind letting me know why this is preferrable?

Reply:
I think the `!=` is a bit clearer (as it is harder to miss the inversion of the logic as it is in the operator rather than a pretty long attribute access away).